docs/en/current/settings/account.md

```diff
@@ -314,7 +314,7 @@ Enable this if you want K-9 Mail to notify you when new (unread) messages have b
 
 ### Notification folders
 K-9 Mail will only create notifications for messages in folders whose 
-[notification class](../folder/#folder-notification-class) matches this setting.
+[notification class](../settings/folder.md#folder-notification-class) matches this setting.
```

Comment:
The URL for this page is https://docs.k9mail.app/en/current/settings/account/
With your change the link would go to https://docs.k9mail.app/en/current/settings/settings/folder.md#folder-notification-clas, wouldn't it?

Reply:
The link did actually work when I tested it, but I've simplified it, thanks for the feedback